src/five/customerize/browser.py

```diff
@@ -27,7 +33,7 @@ def getViews(iface, type=IRequest):
     for reg in gsm.registeredAdapters():
         if (len(reg.required) > 0 and
             reg.required[-1] is not None and
-            reg.required[-1].isOrExtends(type)):
+                reg.required[-1].isOrExtends(type)):
```

Comment:
This extra indent seems strange. But I guess the linter did this automatically. Fine with me.

Reply:
It is suggested by PEP-8 (https://www.python.org/dev/peps/pep-0008/#indentation) to make the lines of the if-statement and the block inside it distinguishable.